src/compass/schemas/s1_cslc_geo.yaml

```diff
@@ -10,12 +10,14 @@ runconfig:
            safe_file_path: list(str(), min=1)
            # Required. List of orbit (EOF) files
            orbit_file_path: list(str(), min=1)
-           # Required. The unique burst ID to process
-           burst_id: str(required=True)
+           # Required. The unique burst IDs to process
+           burst_id: list(str(), min=1, required=False)
```

Comment:
The schema (and default) still contains `output_epsg`, `top_left`, `bottom_right` entries. This should be removed if the database is used